### PR TITLE
Lammps bonds

### PR DESCRIPTION
**Pull request by sudarsan-surendralal**
More functions to enable robust functionality

### CODE REVIEW

File: pyiron/lammps/structure.py [hunk 1]
```diff
@@ -474,20 +474,31 @@ def structure_full(self):
...
-            + "{0} atom types".format(self._structure.get_number_of_species())
+            + "{0} atom types".format(len(sorted_species_list))
             + " \n"
             + "{0} bond types".format(num_bond_types)
             + " \n"
             + "{0} angle types".format(num_angle_types)
             + " \n"
         )
 
+        # atomtypes = list()
```
Comment (jan-janssen) [inline pyiron/lammps/structure.py]: If it is commented out, do we need it?

Comment (sudarsan-surendralal) [reply pyiron/lammps/structure.py]: No we don't. Fixed it here https://github.com/pyiron/pyiron/pull/457/commits/51714a0bd9098cb63844443ccfe19038f6c55e8b

**Comment by jan-janssen** [comment]
Is there any way we can test this functionality?

**Comment by sudarsan-surendralal** [comment]
Shall I merge this @jan-janssen?

**Comment by jan-janssen** [comment]
You can merge it, but still it would be nice to have tests for this.

**Pull request merged**